R/constants-aglu-L2.R

```diff
@@ -0,0 +1,201 @@
+# Column names of tables that are read in to the model interface
+# These correspond to the headers of the same name in the headers/ModelInterface_headers.txt file
+# These are for generic categories; module-specific level2 data names are found in data header files of each module
+
+R <- "region"
+Y <- "year"
+input <- "minicam.energy.input"
```

Comment:
Yeah, I think we will need all of these. We may want a new approach though...there will be a similar, long list for energy.